lebedigital/calibration/posterior_predictive_three_point_bending.py

```diff
@@ -0,0 +1,97 @@
+# third party imports
+import os
+from pathlib import Path
+import random
+import numpy as np
+import matplotlib.pyplot as plt
+import seaborn as sns
+import matplotlib as mpl
+from matplotlib import rc
+import fenics_concrete
+
+# local imports (probeye)
+from probeye.ontology.knowledge_graph_import import import_parameter_samples
+
+# local imports (others)
+from lebedigital.simulation.three_pont_bending_beam import three_point_bending_example
+from lebedigital.calibration.utils import PosteriorPredictive
+from lebedigital.unit_registry import ureg
+
+
+def wrapper_three_point_bending(parameter, known_input):
+    """
+    This is standard template any forward solver needs to be wrapped in
+    Parameters
+    ----------
+    parameter : parameter is E in kN/mm2
+    known_input : Poisson's ratio 'nu'
+
+    Returns
+    -------
+    y : magnitude of stress output from the three point bending test
+
+    """
+    parameters = fenics_concrete.Parameters()
+    parameters['E'] = parameter * 1000 * ureg('N/mm^2')
+    parameters['nu'] = known_input * ureg('')
+
+    y = three_point_bending_example(parameters)
+    return y.magnitude
+
+
+def perform_prediction(forward_solver:callable, parameter:list, known_input:float = 0.2, no_sample :int = 50, mode ='cheap', viz=False):
+    """
+
+    Parameters
+    ----------
+    forward_solver : (callable) The solver through which parametric uncertainty needs to be propagated
+    parameter : (list) the samples of the parameter which was calibrated. (E for eg)
+    known_input: Known input to the solver. (nu for eg)
```

Comment:
if that is supposed to be general, then the known input should be a dict/parameter list. If this is specific to our 3-point bending problem and the known input is nu, then I would just name the parameter nu.